package.json

```diff
@@ -1,124 +1,31 @@
+    "build": "pnpm --filter @ember/string build",
+
+    "start": "concurrently 'npm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
+    "start:addon": "pnpm --filter @ember/string start",
+    "start:test": "pnpm --filter test-app start",
+
+    "lint": "pnpm --filter '*' lint",
+    "lint:fix": "pnpm --filter '*' lint:fix",
+
+    "test": "pnpm --filter '*' test"
   },
-  "devDependencies": {
-    "@babel/cli": "^7.17.6",
-    "@babel/preset-typescript": "^7.16.0",
-    "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.6.0",
-    "@embroider/test-setup": "^0.47.1",
-    "@glimmer/component": "^1.0.4",
-    "@glimmer/tracking": "^1.0.4",
-    "@types/ember": "^3.16.5",
-    "@types/ember-qunit": "^3.4.14",
-    "@types/ember__test-helpers": "^2.0.2",
-    "@types/qunit": "^2.11.2",
-    "@types/rsvp": "^4.0.4",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.29.0",
-    "babel-eslint": "^10.1.0",
-    "broccoli-asset-rev": "^3.0.0",
-    "broccoli-babel-preset-typescript": "^1.0.1",
-    "ember-auto-import": "^2.2.3",
-    "ember-cli": "~3.28.0",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-htmlbars": "^6.0.0",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.2",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.1",
-    "ember-load-initializers": "^2.1.2",
-    "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-page-title": "^6.2.2",
-    "ember-qunit": "^5.1.5",
-    "ember-resolver": "^8.0.3",
-    "ember-source": "~3.28.6",
-    "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^3.12.0",
-    "ember-try": "^1.4.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-ember": "^10.5.7",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-qunit": "^6.2.0",
-    "loader.js": "^4.7.0",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.6.0",
-    "qunit": "^2.17.2",
-    "qunit-dom": "^2.0.0",
-    "release-it": "^14.11.6",
-    "release-it-lerna-changelog": "^3.1.0",
-    "rimraf": "^3.0.2",
-    "typescript": "^4.4.4",
-    "webpack": "^5"
+  "volta": {
+    "node": "18.4.0"
   },
   "engines": {
     "node": "12.* || 14.* || >= 16"
   },
-  "publishConfig": {
-    "registry": "https://registry.npmjs.org"
-  },
-  "ember": {
-    "edition": "octane"
-  },
-  "ember-addon": {
-    "configPath": "tests/dummy/config"
-  },
-  "release-it": {
-    "plugins": {
-      "release-it-lerna-changelog": {
-        "infile": "CHANGELOG.md",
-        "launchEditor": true
-      }
-    },
-    "git": {
-      "tagName": "v${version}"
-    },
-    "github": {
-      "release": true,
-      "tokenRef": "GITHUB_AUTH"
-    },
-    "npm": {
-      "publish": false
+  "pnpm": {
+    "overrides": {
+      "@types/eslint": "7.29.0"
```

Comment:
Did we lose the release-it config? I don't see it in ember-string/package.json either?